packages/components/card/src/card.js

```diff
@@ -8,10 +8,10 @@ const Card = props => (
   <div
     {...filterDataAttributes(props)}
     css={css`
-      display: flex;
-      font-size: 1rem;
-      flex-direction: column;
+      box-sizing: border-box;
```

Comment:
This rule makes it unnecessary to have a wrapper just to set a padding.

Also, without the extra wrapper there's no need for flex here.